src/main/java/org/myrobotlab/service/abstracts/AbstractSpeechSynthesis.java

```diff
@@ -500,6 +517,10 @@ AudioData play(AudioData data, boolean block) {
     if (block) {
       data.mode = AudioData.MODE_BLOCKING;
     }
+    if (audioFile == null) {

```

Comment:
this change is not related to the metadata change.

Reply:
an npe check ?  really?
If it was all converted to do pub/sub properly then this wouldn't be needed, but I can't do all of that at once.
Ok, I can let speech synthesis classes explode until the next pr